src/apps/search-result/search-result.tsx

```diff
@@ -98,6 +99,24 @@ const SearchResult: React.FC<SearchResultProps> = ({ q, pageSize }) => {
     }
   }, [campaignFacets, mutate]);
 
+  // Check for material type filters in url on pageload
+  // (this part will be reworked later based on a change request from the client)
+  useEffect(() => {
+    const searchParams = new URLSearchParams(window.location.search);
+    const materialTypeUrlFilter = searchParams.get("materialType");
+    if (materialTypeUrlFilter) {
+      filterHandler({
+        filterItem: {
+          facet: FacetField.MaterialTypes,
+          term: { key: materialTypeUrlFilter, term: materialTypeUrlFilter }
+        },
+        action: "add"
+      });
+    }
+    // We only want to do this once, so we need the dependency array empty
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
```

Comment:
Consider if this can be replaced by [`useSearchParams()`](https://github.com/streamich/react-use/blob/master/docs/useSearchParam.md) from `react-use`. Perhaps it is too complex compared to what we want to do.

Reply:
Thanks for the suggestion, but it wouldn't save many lines of code. I'll keep it as is :)